### ISSUE TITLE
Severe oligozoospermia

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9800]
**Preferred term label:**
Severe oligozoospermia

**Synonyms**

**Definition (free text, please give PubMed ID)**
>2 - <10 million sperm (>1 - <5 million/mL concentration)

**Parent term (use hpo.jax.org/app)**
HP:0000798

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 399805

**Your nano-attribution (ORCID)**
ORPHA: 399805

**Comment by pnrobinson** [comment]
@MagoWyr 
added with definition

A type of oligozoospermia characterized by a total number of more than 2 but less than 10 million sperm in the ejaculate or a concentration of more than 1 but less than 5 million sperm per milliliter.
